torch/distributed/_tensor/__init__.py

```diff
@@ -111,6 +111,63 @@ def ones(
     )
 
 
+def empty(
+    *size,
+    dtype: Optional[torch.dtype] = None,
+    device: Optional[torch.device] = None,
+    layout: torch.layout = torch.strided,
+    memory_format=torch.contiguous_format,
```

Comment:
I feel we might not need `memory_format` and `pin_memory` exposed, to align more with other tensor constructor signature, and they are rarely used. We can remove these two and see user's feedback